src/frontend/types.ts

```diff
@@ -193,10 +198,7 @@ export interface SettingsContextType {
 }
 
 export interface StoresFilters {
-  legendary: boolean
-  gog: boolean
-  nile: boolean
-  sideload: boolean
+  [key: string]: boolean
 }
```

Comment:
`string` doesn't seem correct here, don't you want `StoreConfigs['filterKey']` here?

Reply:
We could. But that does mean other dynamically added stores will not work. So custom libraries would for instance not work since that dynamically adds multiple stores.

Reply:
But for now I've locked it to `StoreConfigs['filterKey']` since that's a separate MR :)